controllers/clusterurlmonitor/clusterurlmonitor_test.go

```diff
@@ -189,6 +194,186 @@ var _ = Describe("Clusterurlmonitor", func() {
 		})
 	})
 
+	Describe("EnsurePrometheusRuleResourceExists", func() {
+		var (
+			mockStatusWriter     *clientmocks.MockStatusWriter
+			errorToPopulate      error
+			shouldPopulateError  bool
+			deepEqualCalledTimes int
+			deepEqualResponse    bool
+		)
+		BeforeEach(func() {
+			// Why bellow code?
+			// routeMonitorSlo = v1alpha1.SloSpec{}
+			// clusterUrlMonitorSlo = v1alpha1.SloSpec{
+			// 	TargetAvailabilityPercent: "99.95",
+			// }
+			// clusterUrlMonitor.Spec = v1alpha1.clusterUrlMonitor{
+			// 	Slo: clusterUrlMonitorSlo,
+			// }
```

Comment:
is this intended to be in the final version?

if it is then:
`clusterUrlMonitorSlo` is an easy way to modify the clusterUrlMonitor resource without touching it and using muliple `.` notations to get to where the resource is

additionally, this code should be partially in the `JustBeforeEach` to seperate
- default values (BeforeEach)
- assignment to actual object (JustBeforeEach)